docs/views/about.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+About - GOV.UK Prototype kit
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="breadcrumbs">
+        <ol>
+          <li><a href="/docs">GOV.UK Prototype kit</a></li>
+        </ol>
+      </div>
+      <h1 class="heading-xlarge">About</h1>
+
+      <p class="lede">
+        The prototype kit provides a simple way to make interactive prototypes that look like pages on GOV.UK. These prototypes can be used to show ideas to people you work with, and to do user research
+      </p>
+
+      <h2 class="heading-medium">Community</h2>
+
+      <p>We have two Slack channels for the Prototype kit. You'll need a government email address to join.</p>
+      <ul class="list list-bullet">
+        <li><a href="https://ukgovernmentdigital.slack.com/messages/prototype-kit/">Slack channel for users of the prototype kit</a></li>
+        <li><a href="https://ukgovernmentdigital.slack.com/messages/prototype-kit-dev/">Slack channel for developers of the prototype kit</a></li>
+      </ul>
+
+      
+      <h2 class="heading-medium">Principles</h2>
+
+      <ul class="list list-bullet">
```

Comment:
The list needs to be preceded by "The prototype kit:" for the points to be grammatically correct